js/ui/shellDBus.js

```diff
@@ -516,3 +517,107 @@ var AppStoreService = class {
         this._dbusImpl.emit_signal('ApplicationsChanged', GLib.Variant.new('(as)', [allApps]));
     }
 };
+
+const HackableAppIface = loadInterfaceXML('com.endlessm.HackableApp');
+var HackableApp = class {
+    constructor(manager, session) {
+        this._dbusImpl = Gio.DBusExportedObject.wrapJSObject(HackableAppIface, this);
+        Gio.bus_own_name_on_connection(Gio.DBus.session, 'com.endlessm.HackableApp',
```

Comment:
It allowed me to do something like
`gdbus introspect --session --dest com.endlessm.HackableApp --object-path /com/endlessm/HackableApp/1 --only-properties`

instead of using the org.gnome.Shell dbus name. Any suggestion here?

Reply:
Why is it a problem for that use case to use the `org.gnome.Shell` dbus name?

Reply:
Nothing... probably just some wrong supposition. I'll remove this then.